### PR TITLE
translated page add-404-page

### PR DESCRIPTION
**Pull request by spielers**
please merge this page

### CODE REVIEW

**Comment by spielers** [comment]
sorry for concern I separately added page and translated.

**Comment by spielers** [comment]
Sir please review this and  merge this .

File: docs/docs/add-404-page.md [hunk 1]
```diff
@@ -1,28 +1,19 @@
 ---
-title: Adding a 404 Page
+title: 404 पेज ऐड करना
 ---
 
-To create a 404 page create a page whose path matches the regex
-`^\/?404\/?$` (`/404/`, `/404`, `404/` or `404`). Most often you'll want to create a React component page at
-`src/pages/404.js`.
+404 पेज बनाने के लिए एक पेज बनाये जिसका पाथ `^\/?404\/?$` (`/404/`, `/404`, `404/` or `404`) regex से मेल खाता हो। ज़्यादातर आप एक React कौम्पोनॅन्ट पेज बनाना चाहेंगे `src/pages/404.js`।.
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]:
```
चाहेंगे `src/pages/404.js`।.
```
=>
```
चाहेंगे `src/pages/404.js`।
```

there is an unwanted dot at the end of the line

File: docs/docs/add-404-page.md [hunk 2]
```diff
@@ -1,28 +1,19 @@
...
 
-Gatsby ensures that your 404 page is built as `404.html` as many static hosting
-platforms default to using this as your 404 error page. If you're hosting your
-site another way, you'll need to set up a custom rule to serve this file for 404
-errors.
+Gatsby यह सुनिश्चित करता है कि आपका 404 `404.html` की तरह बना हो क्यूंकि ज़्यादातर स्टेटिक होस्टिंग प्लेटफॉर्म्स इसीको डिफ़ॉल्ट रूप से 404 एरर पेज की तरह इस्तेमाल करते है। यदि आप अपनी साइट को दूसरे तरीके से होस्ट कर रहे हैं, तो आपको 404 एर्ररस के लिए इस फ़ाइल को सर्व करने के लिए एक कस्टम नियम स्थापित करने की आवश्यकता होगी।
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `आपका 404` => `आपका 404 पेज`

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `स्थापित` => `सेट-अप`

File: docs/docs/add-404-page.md [hunk 3]
```diff
@@ -1,28 +1,19 @@
...
 
-Because Gatsby creates this page for you by default, there is no need to configure
-it in your `gatsby-node.js` file.
+क्योंकि Gatsby डिफ़ॉल्ट रूप से आपके लिए यह पेज बनाता है, इसलिए इसे आपकी `gatsby-node.js` फ़ाइल में कॉन्फ़िगर करने की कोई आवश्यकता नहीं है।
 
-When developing using `gatsby develop`, Gatsby uses a default 404 page that
-overrides your custom 404 page. However, you can still preview your 404 page by
-clicking "Preview custom 404 page" to verify that it's working as expected. This is
-useful when you're developing so that you can see all the available pages.
+प्रयोग करते समय `gatsby develop`, Gatsby एक डिफ़ॉल्ट 404 पेज का उपयोग करता है जो आपके कस्टम 404 पृष्ठ को ओवरराइड करता है। हालाँकि, आप अभी भी अपने 404 पेज का पूर्वावलोकन कर सकते हैं "कस्टम 404 पेज का पूर्वावलोकन करें" पर क्लिक करके यह सत्यापित करें कि यह अपेक्षित है। यह उपयोगी है जब आप विकसित कर रहे हैं ताकि आप सभी उपलब्ध पेजेज को देख सकें।
```
Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `पृष्ठ` => `पेज`

Comment (arshadkazmi42) [inline docs/docs/add-404-page.md]: `पूर्वावलोकन` => `प्रीव्यू`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work.
Just a couple of fixes and this is good to go

**Comment by spielers** [comment]
errors fixed just merge this thing, sir I will start working on other

**Comment by arshadkazmi42** [comment]
@spielers Please don't hurry. We want to do the translation, but in the right way. 
I am the only reviewer available. Sometimes it takes some time to get to review.
Hope you understand.

**Review by arshadkazmi42 — APPROVED** [review]
Looks good.
Thank you for working on this.

**Comment by arshadkazmi42** [comment]
@spielers Congrats on landing your first PR.
You can claim your swag from above link. Thank you for your contribution

**Pull request merged**